core/drivers/crypto/caam/cipher/local.h

```diff
@@ -44,8 +53,16 @@ struct cipherdata {
 	struct caambuf ctx;          /* CAAM Context Register */
 	struct caamblock blockbuf;   /* Temporary Block buffer */
 	const struct cipheralg *alg; /* Reference to the algo constants */
+
+	/* Additionnal Data for the MAC */
+	unsigned int mode; /* MAC Mode */
```

Comment:
Would it be possible to use an enum instead? If not, at least comment which defines are supposed to be used.

Reply:
Added ```/* MAC TEE_CHAIN_MODE* */```